templates/zerver/integrations/errbot.md

```diff
@@ -25,23 +27,42 @@ Run your favorite chatbot in Zulip!
         BOT_LOG_FILE = r'<path/to/your/errbot/logfile.log>'
         BOT_LOG_LEVEL = logging.INFO
 
-        BOT_IDENTITY = {  # Fill this with the corresponding values in your bot's `.zuliprc`
-          'email': '<err-bot@your.zulip.server>',
+        BOT_IDENTITY = {
+          'email': '<your-bot-name@yourZulipDomain.zulipchat.com>',
           'key': '<abcdefghijklmnopqrstuvwxyz123456>',
-          'site': '<http://your.zulip.server>'
+          'site': '<https://yourZulipDomain.zulipchat.com>'
         }
-        BOT_ADMINS = ('<your@email.address',)
+        BOT_ADMINS = ('<your@email.address>',)
         CHATROOM_PRESENCE = ()
-        BOT_PREFIX = '<@**err-bot**>'  # Providing errbot's full name in Zulip lets it respond to @-mentions.
+        BOT_PREFIX = '@**<your-bot-name>**'
```

Comment:
Changed the formatting of this line. Since we want the user to replace the entire part in the angular brackets, moved them inside the user-tag syntax. And switched the bot name to a generic instruction, so that it's clear that it's meant to be replaced.
Removed the comment because the clarification of purpose is not necessary here.